lib/signaling/v2/cancelableroomsignalingpromise.js

```diff
@@ -136,13 +137,12 @@ function createCancelableRoomSignalingPromise(token, wsServer, localParticipant,
     }
   });
 
-  cancelablePromise.catch(error => {
+  cancelablePromise.catch(() => {
     if (transport) {
       transport.disconnect();
       transport = null;
     }
     peerConnectionManager.close();
-    throw error;
```

Comment:
if not throw at least we should log the error here.

Reply:
@makarandp0 We already log this in several other places.